packages/ios-client/Sources/FishjamClient/FishjamClientInternal.swift

```diff
@@ -439,12 +439,12 @@ internal class FishjamClientInternal: WebSocketDelegate, PeerConnectionListener,
     func onTracksAdded(endpointId: String, tracks: [String: TrackData]) {
         if localEndpoint.id == endpointId { return }
 
-        guard let endpoint = remoteEndpointsMap.removeValue(forKey: endpointId) else {
+        guard let endpoint = remoteEndpointsMap[endpointId] else {
             sdkLogger.error("Failed to process TracksAdded event: Endpoint not found: \(endpointId)")
             return
         }
 
-        var updatedTracks = [String: Track]()
+        var updatedTracks: [String: Track] = endpoint.tracks
```

Comment:
Do you still need type declaration?

Reply:
no
but i do not consider it a bad habbit

Reply:
Sure! My point is that it doesn't provide any more information and you can always setup your IDE to hint types if you like.

What do you think we would commit to a predefined style guide?

Reply:
Predefined style guide it is a great idea!
Lets create task for it and apply it the future.

Reply:
Great! 
https://linear.app/swmansion/issue/FCE-492/style-guide-for-iosswift